src/main/java/com/mitchej123/hodgepodge/mixins/early/fml/MixinEventBus.java

```diff
@@ -0,0 +1,24 @@
+package com.mitchej123.hodgepodge.mixins.early.fml;
+
+import java.util.Map;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import cpw.mods.fml.common.ModContainer;
+import cpw.mods.fml.common.eventhandler.EventBus;
+
+@Mixin(value = EventBus.class, remap = false)
+public class MixinEventBus {
```

Comment:
this mixin is actually not needed, the map is using WeakReference for the keys and values